plugins/clamd.js

```diff
@@ -16,49 +16,49 @@ exports.load_excludes = function() {
     for (var i=0; i < list.length; i++) {
         var re;
         switch (list[i][0]) {
-        case '!':
-
-            if (list[i][1] === '/') {
-                // Regexp exclude
+            case '!':
+
+                if (list[i][1] === '/') {
+                    // Regexp exclude
+                    try {
+                        re = new RegExp(list[i].substr(2, list[i].length-2),'i');
+                        new_skip_list_exclude.push(re);
+                    }
+                    catch (e) {
+                        plugin.logerror(e.message + ' (entry: ' + list[i] + ')');
+                    }
+                }
+                else {
+                    // Wildcard exclude
+                    try {
+                        re = new RegExp(
+                                utils.wildcard_to_regexp(list[i].substr(1)),'i');
+                        new_skip_list_exclude.push(re);
+                    }
+                    catch (e) {
+                        plugin.logerror(e.message + ' (entry: ' + list[i] + ')');
+                    }
+                }
+                break;
+            case '/':
+                // Regexp skip
                 try {
-                    re = new RegExp(list[i].substr(2, list[i].length-2),'i');
-                    new_skip_list_exclude.push(re);
+                    re = new RegExp(list[i].substr(1, list[i].length-2),'i');
+                    new_skip_list.push(re);
```

Comment:
This really is **only** a whitelist change. If you use side-by-side diffs with a really wide screen it makes the change easier to see.

This does raise an interesting question. To indent case or not?  I prefer no but I chose yes, based on **most** of the `case` calls being already indented.